nbstripout/_nbstripout.py

```diff
@@ -471,7 +473,9 @@ def main():
                     warnings.simplefilter("ignore", category=UserWarning)
                     nb = read(f, as_version=NO_CONVERT)
 
-            nb = strip_output(nb, args.keep_output, args.keep_count, extra_keys, args.strip_empty_cells, _parse_size(args.max_size))
```

Comment:
BTW: The args.strip_init_cells option is missing in the original code. I followed alone that line.